### PR TITLE
List role rolebinding

### LINKED ISSUE #41: Add keywords for roles and role bindings
keywords could be implemented based on the kubernetes client functions for 
[list_namespaced_role](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/RbacAuthorizationV1Api.md#list_namespaced_role)
and
[list_namespaced_role_binding](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/RbacAuthorizationV1Api.md#list_namespaced_role_binding)

### PR DESCRIPTION
**Pull request by satish-nubolab**
Add keyword for list of role and rolebinding.

Fixes #41

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Keyword for list ingress [#38](https://github.com/devopsspiral/KubeLibrary/pull/38) by [@satish-nubolab](https://github.com/satish-nubolab)
 - Keyword for list cronjob [#48](https://github.com/devopsspiral/KubeLibrary/pull/48) by [@satish-nubolab](https://github.com/satish-nubolab)
 - Keyword for list daemonset [#50](https://github.com/devopsspiral/KubeLibrary/pull/50) by [@satish-nubolab](https://github.com/satish-nubolab)
+- Keyword for getiing role and rolebinding [#56](https://github.com/devopsspiral/KubeLibrary/pull/56) by [@satish-nubolab](https://github.com/satish-nubolab)
```
Comment (m-wcislo) [inline CHANGELOG.md:20]: Please add it to *In progress* section we had a new release recently, so this change will be part of next release

File: src/KubeLibrary/KubeLibrary.py [hunk 2]
```diff
@@ -78,13 +78,13 @@ def reload_config(self, kube_config=None, context=None, incluster=False, cert_va
...
             except TypeError:
                 logger.error('Neither KUBECONFIG nor ~/.kube/config available.')
-
         self._add_api('v1', client.CoreV1Api)
         self._add_api('extensionsv1beta1', client.ExtensionsV1beta1Api)
         self._add_api('batchv1', client.BatchV1Api)
         self._add_api('appsv1', client.AppsV1Api)
         self._add_api('batchv1_beta1', client.BatchV1beta1Api)
         self._add_api('custom_object', client.CustomObjectsApi)
+        self._add_api('rbac_authv1_api', client.RbacAuthorizationV1Api)
```
Comment (m-wcislo) [inline src/KubeLibrary/KubeLibrary.py:87]: Please add the new api in https://github.com/devopsspiral/KubeLibrary/blob/6215736fbc092a51ac194f4dcefaa8296811cb5b/test/test_KubeLibrary.py#L91 to cover it with tests.

File: src/KubeLibrary/KubeLibrary.py [hunk 3]
```diff
@@ -641,6 +641,28 @@ def get_daemonset_details_in_namespace(self, name, namespace):
         ret = self.appsv1.read_namespaced_daemon_set(name, namespace)
         return ret
 
+    def get_role_in_namespace(self, namespace):
```
Comment (m-wcislo) [inline src/KubeLibrary/KubeLibrary.py:644]: Better to call it *get_roles_in_namespace* as the returning value is list. Same for role binding**s**

**Review by m-wcislo — CHANGES_REQUESTED** [review]
Would be nice to change the coverage threshold in https://github.com/devopsspiral/KubeLibrary/blob/6215736fbc092a51ac194f4dcefaa8296811cb5b/.coveragerc#L6. If you click on CI results there is job for lint-and-coverage you can take the coverage from there and set it in the mentioned file.

File: CHANGELOG.md (hunk 1 above)
Comment (satish-nubolab) [reply CHANGELOG.md]: Done.

File: src/KubeLibrary/KubeLibrary.py (hunk 2 above)
Comment (satish-nubolab) [reply src/KubeLibrary/KubeLibrary.py:87]: Done.

File: src/KubeLibrary/KubeLibrary.py (hunk 3 above)
Comment (satish-nubolab) [reply src/KubeLibrary/KubeLibrary.py]: Done.

**Review by m-wcislo — APPROVED** [review]

**Comment by m-wcislo** [comment]
Nice! Merging.

**Pull request merged**